### DISCUSSION TITLE
Can HorizontalBox not affect background colour?

### DISCUSSION THREAD

**Discussion by git-vcs** [patrykandpatrick/vico#964] [Questions]
### Question

From example code Chart 6 I wonder, is there a way to make the HorizontalBox only colour the bar graph and not the background? 
I was hoping to use something like this to put different colours inside different y-ranges on a cartesian line chart. Also the line colour of the graph should not be effected. Perhaps there is a better way than using HorizontalBox?

something like this:

![Untitled](https://github.com/user-attachments/assets/55d506a2-299f-49a5-a40b-85a9cd0f933c)

also: I apologize in advance if this question closely resembles https://github.com/patrykandpatrick/vico/discussions/959 🙏 

### Vico version(s)

2.0.0-beta3

### UI framework(s)

Views

**Comment by Gowsky** [comment]
Hello, @git-vcs. Yes, it's possible. Please see [here](https://github.com/patrykandpatrick/vico/discussions/483#discussioncomment-10858540).
